lib/phoenix_live_view/debug.ex

```diff
@@ -2,55 +2,100 @@ defmodule Phoenix.LiveView.Debug do
   @moduledoc """
   Functions for runtime introspection and debugging of LiveViews.
 
-  TODO
+  This module provides utilities for inspecting and debugging LiveView processes
+  at runtime. It allows you to:
+
+    * List all currently connected LiveViews
+    * Check if a process is a LiveView
+    * Get the socket of a LiveView process
+    * Inspect LiveComponents rendered in a LiveView
+
+  ## Examples
+
+      # List all LiveViews
+      iex> Phoenix.LiveView.Debug.list_liveviews()
+      [%{pid: #PID<0.123.0>, view: MyAppWeb.PostLive.Index, topic: "lv:12345678", transport_pid: #PID<0.122.0>}]
+
+      # Check if a process is a LiveView
+      iex> Phoenix.LiveView.Debug.liveview_process?(pid(0,123,0))
+      true
+
+      # Get the socket of a LiveView process
+      iex> Phoenix.LiveView.Debug.socket(pid(0,123,0))
+      {:ok, %Phoenix.LiveView.Socket{...}}
+
+      # Get information about LiveComponents
+      iex> Phoenix.LiveView.Debug.live_components(pid(0,123,0))
+      {:ok, [%{id: "component-1", module: MyAppWeb.PostLive.Index.Component1, ...}]}
+
   """
 
   @doc """
-  Returns a list of all currently connected LiveView processes (on the curret node).
+  Returns a list of all currently connected LiveView processes (on the current node).
+
+  Each entry is a map with the following keys:
+
+    - `pid`: The PID of the LiveView process.
+    - `view`: The module of the LiveView.
+    - `topic`: The topic of the LiveView's channel.
+    - `transport_pid`: The PID of the transport process.
+
+  The `transport_pid` can be used to group LiveViews on the same page.
 
   ## Examples
 
-      iex> list_liveview_processes()
-      [#PID<0.123.0>]
+      iex> list_liveviews()
+      [%{pid: #PID<0.123.0>, view: MyAppWeb.PostLive.Index, topic: "lv:12345678", transport_pid: #PID<0.122.0>}]
 
   """
-  def list_liveview_processes do
-    for pid <- Process.list(), liveview_process?(pid) do
-      pid
+  def list_liveviews do
+    for pid <- Process.list(), dict = lv_process_dict(pid), not is_nil(dict) do
```

Comment:
And are we adding `list_liveviews_pids`?

Reply:
Ah, nevermind, this one is cheap enough that we don't need the pids version. I like it. Ship it!